core/storage/suggestion/gae_models.py

```diff
@@ -3427,3 +3429,95 @@ def export_data(
                     model.last_contribution_date.isoformat())
             }
         return user_data
+
+
+class TranslationCoordinatorsModel(base_models.BaseModel):
+    """Storage model for rights related to translation coordinator.
+
+    The id of each instance is the id of the corresponding language.
+    The ISO 639-1 language code.
```

Comment:
This is a sentence fragment ... I don't really understand it. Can you write this docstring more coherently?

Reply:
PTAL again. Does it look good now?